project/Merging.scala

```diff
@@ -9,6 +9,7 @@ object Merging {
     case PathList("com", "typesafe", _ @ _*) => MergeStrategy.last
     case PathList("com", "google", "auto", "value", _ @ _*) => MergeStrategy.last
     case PathList("io", "sundr", _ @ _*) => MergeStrategy.last
+    case PathList("javax", "activation", _ @ _*) => MergeStrategy.first
```

Comment:
This comes in through clients built with swagger codegen. I'm not positive that this is the most correct merge strategy, but I don't _think_ it matters hugely.

Reply:
I much prefer using exclude rules instead of the blunt-intstrument merge strategy. Do you know where the conflicts are, and could you add excludes for those instead?

fwiw, we recently also had assembly issues in orch on javax.activation, though that was via javax.mail: broadinstitute/firecloud-orchestration#838

Reply:
I didn't experiment with that, I think I'd need some help to try it.